paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -151,37 +151,40 @@ internal fun Wallet(
     val padding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
     if (containerState.shouldShow) {
-        Column(modifier = modifier.padding(horizontal = padding)) {
-            if (containerState.showGooglePay) {
-                GooglePayButton(
-                    state = googlePayButtonState?.convert(),
-                    isEnabled = buttonsEnabled,
-                    onPressed = viewModel::checkoutWithGooglePay,
-                    modifier = Modifier.padding(top = 7.dp),
-                )
+        Column {
+            Column(modifier = modifier.padding(horizontal = 4.dp)) {
```

Comment:
Can we achieve the padding without the additional `Column`, e.g. by setting it on the `modifier` that we pass into `GooglePayButton`?